std/special/compiler_rt/floatdidf_test.zig

```diff
@@ -0,0 +1,53 @@
+const __floatdidf = @import("floatdidf.zig").__floatdidf;
+const testing = @import("std").testing;
+
+fn test__floatdidf(a: i64, expected: f64) void {
+    const r = __floatdidf(a);
+    testing.expect(r == expected);
+}
+
+test "floatdidf" {
```

Comment:
Why is this test in a separate file? While e.g. `floatsiXf` has tests in the same file?

Reply:
There's a precedent for having them in a separate file to save space of the shipped .zig files. However it's not required.

Reply:
I don't mind either way, I just think it should be consistent.

Reply:
I try to put long test cases in a separate file and keep short ones at the bottom of the file where the implementation is.